src/@types/aquarius/BaseQueryParams.d.ts

```diff
@@ -11,6 +26,7 @@ interface BaseQueryParams {
   sortOptions?: SortOptions
   aggs?: any
   filters?: FilterTerm[]
+  bool?: boolFilter[]
```

Comment:
suggestion(blocking): Use a const name like `boolFilter` instead of generic `bool`.

Reply:
this is a requirement from elasticsearch, as they expect this to be named `"bool"` in the request body